includes/Core/Key_Metrics/Key_Metrics.php

```diff
@@ -82,12 +82,14 @@ public function register() {
 	 * Adds the status of the Key Metrics widget setup to the inline JS data.
 	 *
 	 * @since 1.108.0
+	 * @since n.e.x.t Include keyMetricsSetupCompletedByUserID
 	 *
 	 * @param array $data Inline JS data.
 	 * @return array Filtered $data.
 	 */
 	private function inline_js_base_data( $data ) {
-		$data['keyMetricsSetupCompleted'] = (bool) $this->key_metrics_setup_completed->get();
+		$data['keyMetricsSetupCompleted']         = (bool) $this->key_metrics_setup_completed->get();
```

Comment:
As mentioned above, we can do the same thing via a selector from the ID value – I don't see a reason to pass this along from the server as a separate bit of state.

Reply:
@aaemnnosttv This is also according to the IB, to include `keyMetricsSetupCompletedByUserID` for getting user ID and keep the current `keyMetricsSetupCompleted` as boolean

Reply:
See above https://github.com/google/site-kit-wp/pull/7661#discussion_r1346295245